Jenkinsfile.release

```diff
@@ -0,0 +1,90 @@
+            string(defaultValue: 'Staging', description: 'Environment name, used to target Batch Compute Environments.', name: 'RF_DEPLOYMENT_ENVIRONMENT', trim: false),
+            string(defaultValue: 'develop', description: 'Branch of azavea/raster-foundry-deployment used for deployment.', name: 'RF_DEPLOYMENT_BRANCH', trim: false)]
+        )]
+    )
+    
+    try{
+        env.AWS_DEFAULT_REGION = "us-east-1"
+
+        stage('checkout') {
+          checkout([$class: 'GitSCM',
+                   branches: [[name: params.GIT_COMMIT ]],
+                   extensions: [[$class: 'PruneStaleBranch']],
+                   userRemoteConfigs: [[credentialsId: '3bc1e878-814a-43d1-864e-2e378ebddb0f',
+                                        url: 'https://github.com/raster-foundry/raster-foundry',
+                                        ]]
+                     ])
+        }
+
+        stage('re-tag images'){
+            sh 'eval $(aws ecr get-login --no-include-email)'
+            withCredentials([[$class: 'StringBinding',
+                              credentialsId: 'AWS_ECR_ENDPOINT',
+                              variable: 'AWS_ECR_ENDPOINT']]) {
+              wrap([$class: 'AnsiColorBuildWrapper']) {
+                // Retag images with release tag and push to ECR
+                [
+                    "raster-foundry-nginx-api",
+                    "raster-foundry-nginx-tiler",
+                    "raster-foundry-api-server",
+                    "raster-foundry-tile-server",
+                    "raster-foundry-batch",
+                    "raster-foundry-migrations"
+                ].each { image_url ->
+                    retagImage(env.AWS_ECR_ENDPOINT + "/${image_url}", params.GIT_COMMIT, params.RELEASE_TAG)
+                }
+              }
+            }
+        }
+
+        stage('deploy docs'){
+          sh './scripts/deploy-docs'
+        }
+
+        stage('infra') {
+
+            checkout scm: [$class: 'GitSCM',
+                           branches: [[name: params.RF_DEPLOYMENT_BRANCH]],
+                           extensions: [[$class: 'RelativeTargetDirectory',
+                                         relativeTargetDir: 'raster-foundry-deployment']],
+                           userRemoteConfigs: [[credentialsId: '3bc1e878-814a-43d1-864e-2e378ebddb0f',
+                                                url: 'https://github.com/azavea/raster-foundry-deployment.git']]]
+
+            def slackMessage = ":rasterfoundry: production deployment in progress"
+            slackSend color: 'warning', message: slackMessage, channel: "@tnation"
+
+            dir('raster-foundry-deployment') {
+              wrap([$class: 'AnsiColorBuildWrapper']) {
+                sh 'docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm terraform ./scripts/infra plan'
```

Comment:
An apply needs to go here too right?

Reply:
Yeah, I disabled it for tests because I didn't want to clobber what was already up on staging.